test_env.py

```diff
@@ -0,0 +1,45 @@
+import os
+
+# The test scripts accept 'stable' for ASTROPY_VERSION to test that it's
+# properly parsed hard-wire the latest stable branch version here
+
+LATEST_ASTROPY_STABLE = '1.0'
+
+if os.environ.get('PIP_DEPENDENCIES', None) is not None:
+    PIP_DEPENDENCIES = os.environ['PIP_DEPENDENCIES'].split(' ')
+else:
+    PIP_DEPENDENCIES = []
+
+if os.environ.get('CONDA_DEPENDENCIES', None) is not None:
+    CONDA_DEPENDENCIES = os.environ['CONDA_DEPENDENCIES'].split(' ')
+else:
+    CONDA_DEPENDENCIES = []
+
+dependency_list = PIP_DEPENDENCIES + CONDA_DEPENDENCIES
+
+
+def test_numpy():
+    if os.environ.get('NUMPY_VERSION', None) is not None:
```

Comment:
I know I suggested this, but in fact you could just do ``if 'NUMPY_VERSION' in os.environ``